Makefile

```
@@ -0,0 +1,11 @@
+venv: requirements.txt requirements-dev.txt requirements-test.txt
+	python -m venv venv
+
+	. venv/bin/activate; \
+	pip install -r requirements.txt; \
+	pip install -r requirements-dev.txt; \
+	pip install -r requirements-test.txt; \
+	pip install -e .
+
+	pre-commit install
+	pre-commit run
```

Didn't run for me ... had to update like below and that ran 

```suggestion
	pip install -e .; \
	pre-commit install; \
	pre-commit run; \
```